### ISSUE TITLE
Audio issues and game stutters with widescreen fix enabled

### ISSUE THREAD

**Issue by darkelf65** [elishacloud/Silent-Hill-2-Enhancements#369]
After I added all enhanced edition files in the game directory and launched the game, since cutscenes like the trailers' ones and the bathroom at the start kept stutter and shortly freezing after few seconds ( the audio too) I disabled d3d8to9 option and now they run fine, but the actual gameplay parts are laggy, the audio is quite popping and twisted and the game in general is choppy and slow. I found out that disabling the widescreen fix solves this by a lot, but I have to play it on a very small window at the center of my screen. How can I play fullscreen without having to deal with constant lag and bad audio?

**Comment by Polymega** [comment]
Hello, 

Please upload a copy of your `d3d8.log` file.

**Comment by darkelf65** [comment]
[d3d8.log](https://github.com/elishacloud/Silent-Hill-2-Enhancements/files/5857690/d3d8.log)

**Comment by r2rX** [comment]
@darkelf65 From a quick glance at the log, it seems that the drivers for your GPU aren't currently installed (due to Windows using the 'Microsoft Basic Display Driver'). Try install the latest driver for your GPU (whether it's Intel and/or NVIDIA) and try test again.

**Comment by Polymega** [comment]
Yes, looking at your log, you're using a laptop. Can you give us the exact model of your Sony laptop? Perhaps we could help determine what graphics type the laptop should have/be using.

**Comment by darkelf65** [comment]
My Vaio Sony computer model is SVF153A1YM. I'm trying to download a newer Nvidia driver as r2rX suggested

**Comment by r2rX** [comment]
According to the log, it seems to be the Sony VAIO SV-F1532M1EW model, which would be an Intel HD Graphics 2000/4000 and/or NVIDIA GT 735/740M.

The general specs of this model series are: https://www.notebookcheck.net/Sony-Vaio-SV-F-Series.92403.0.html

**Comment by r2rX** [comment]
> 
> 
> My Vaio Sony computer model is SVF153A1YM. I'm trying to download a newer Nvidia driver as r2rX suggested

If your laptop has a hybrid GPU configuration (both Intel HD Graphics & NVIDIA), make sure both are updated (always a good idea). Assuming the hybrid setup, make sure to run the Silent Hill 2 executable with the NVIDIA GPU.

**Comment by darkelf65** [comment]
Yes I disabled the Intel driver and kept using the Nvidia's one months ago to not create interferences by using both at the same time

**Comment by Polymega** [comment]
> My Vaio Sony computer model is SVF153A1YM

Hm, I couldn't easily find info/specs on this model.

But if your laptop uses both an IGP and dedicated graphics processor (Nvidia as you said), you can use this guide to ensure SH2 PC is using your Nvidia card and not the IGP: https://www.techadvisor.co.uk/how-to/pc-components/how-set-default-graphics-card-3612668/

**Comment by r2rX** [comment]
> 
> 
> > My Vaio Sony computer model is SVF153A1YM
> 
> Hm, I couldn't easily find info/specs on this model.

Neither could I. The closest is a general spec sheet for the SV-F series: https://www.notebookcheck.net/Sony-Vaio-SV-F-Series.92403.0.html

**Comment by Polymega** [comment]
There's a chance your laptop only has an integrated graphics processor (IGP), and depending on how weak/old the IGP is, it will definitely stutter and lag depending on the resolution you're playing the game at or if using demanding features of the project such as the Fog Fix, built-in SMAA filter, or HD fonts/images.

Some IGPs can handle a lot of the graphical improvements we've done for the game, but many can't (especially ones made in 2011 or earlier). This is why, [on our recommended specs section](http://enhanced.townofsilenthill.com/SH2/#systemreq), we only recommend a dedicated graphics card for playing.

You can try disabling the Fog Fix and HD fonts within the `d3d8.ini` file and playing the game at a lower resolution. That might help some. To take it even more extreme, you can delete/remove the high resolution FMVs and full screen images from our project found within the `\sh2e\` folder. By doing this you'd be missing a lot of the improvements we have to offer, but it might be enough to get the game running more stable.

**Comment by darkelf65** [comment]
Yeah, tt was the dedicated graphics card in the end. I followed that guide and changed preferred graphic card from IGC to high performance Nvidia driver in the Nvidia control pannel. I had some headaches tough, because I couldn't enter the control pannel and I found out after about 2 hours I had to reanable the Intel driver...very odd but it did the trick. Now the game runs flawless and I can finally enjoy these super-clean textures. I also reanabled the widescreen fix along with the fog fix. I will also try to reanable the d3d8to9 option. However, the d3d8 file says it is using the Intel driver instead of the Nvidia driver, strange, but the game is perfect anyway. Thanks to you both!
